src/doc/src/commands/cargo-add.md

```diff
@@ -4,7 +4,7 @@
 
 ## NAME
 
-cargo-add - Add dependencies to a Cargo.toml manifest file
+cargo-add --- Add dependencies to a Cargo.toml manifest file
```

Comment:
Example for what I meant. This doesn't become an em dash, while other characters and HTML entities in the same file become fancy. Looks like this would be renered incorrectly downstream, if Markdown parsers out of the scope of this repo don't provide an additional layer of smart-punctuation processing.

Same for other files in same directory.